composer.json

```diff
@@ -53,7 +53,7 @@
         "aws/aws-sdk-php": "^2.7",
         "jackalope/jackalope-doctrine-dbal": "^1.1",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
-        "friendsofsymfony/rest-bundle": "^1.7.4 || ^2.0",
+        "friendsofsymfony/rest-bundle": "^2.0",
```

Comment:
I understand, so do two different versions Tags. Or fix it differently, but in all case your library is not compatible with 2.0 that you was targetting ;-)